com.stansassets.build/Editor/Steps/UnityPlayerBuildStep.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEditor.Callbacks;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace StansAssets.Build.Editor
+{
+    class UnityPlayerBuildStep : IBuildStep
+    {
+        public int Priority => m_Priority;
+
+        public string Name => m_Name;
+        private string m_Name = "StepName";
```

Comment:
1. Why we need field and property here instead of just a single property?
2. Why UnityPlayerBuildStep has "StepName" name?